### PR TITLE
update getting started to include ios library import

### PR DESCRIPTION
**Pull request by bell-steven**
As per [this issue](https://github.com/kmagiera/react-native-gesture-handler/issues/320#issuecomment-557108570).

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -107,14 +107,18 @@ Make sure to [update the React Native CLI to the latest version](https://github.
...
 
-There is no additional configuration required on iOS except what follows in the next steps.
-
 If you're in a CocoaPods project (the default setup since React Native 0.60), make sure to install pods before you run your app:
 
 ```sh
 cd ios && pod install
 ```
 
+For React Native 0.60 or greater, add the library as the first import in your index.js file:
```
Comment (tomtargosz) [inline docs/getting-started.md]: The issue is happening on versions >= `0.61.0`.

Comment (bell-steven) [reply docs/getting-started.md]: whoops. Fixed. Thank you.

**Comment by tomtargosz** [comment]
@kmagiera what are your thoughts on this? I know some devs (my team included) haven't gone this route because we weren't sure if it was a "quick hack" or a solid solution to the issue we've been facing. If this is the recommended solution it'd be great to get it added to the wiki to clear up any confusion.

**Comment by osdnk** [comment]
Thanks!

**Pull request merged**